tests/src/test/scala/common/rest/WskRest.scala

```diff
@@ -85,15 +81,42 @@ import common.WskActorSystem
 import common.WskProps
 import whisk.core.entity.ByteSize
 import whisk.utils.retry
-import javax.net.ssl.{HostnameVerifier, KeyManager, SSLContext, SSLSession, X509TrustManager}
+import javax.net.ssl._
+
+import akka.actor.ActorSystem
 import com.typesafe.sslconfig.akka.AkkaSSLConfig
 import java.nio.charset.StandardCharsets
+import java.security.KeyStore
 
 class AcceptAllHostNameVerifier extends HostnameVerifier {
   override def verify(s: String, sslSession: SSLSession): Boolean = true
 }
 
 object SSL {
+
+  val controllerProtocol = WhiskProperties.getProperty("controller.protocol")
+  val controllerKeyStorePath = WhiskProperties.getProperty("controller.ssl.keystore.path")
+  val controllerKeyStorePassword = WhiskProperties.getProperty("controller.ssl.keystore.password")
+  val controllerKeyStoreFlavor = WhiskProperties.getProperty("controller.ssl.keystore.flavor")
+  val controllerClientAuth = WhiskProperties.getProperty("controller.ssl.clientAuth").toBoolean
```

Comment:
You already put these variables into `application.conf`. Can we use them instead?

Afterwards you can remove the entries from `whisk.properties` again.

Reply:
Please refer to https://github.com/apache/incubator-openwhisk/pull/3111/commits/04579a3b786c582d0ff35f679208e63069473b68 for our future path of solving this. I back that we should move these to pureconfig and write the `.conf` the same way we write whisk.properties today.